SU-SWS/stanford_profile
# D8CORE-1177: override wysiwyg text paragraph styles

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-1177_

# Needed Review By (Date)
- Sprint end

# Urgency
- For RC1

# Steps to Test

- Checkout this branch
- Navigate to a page with WYSIWYG text
- Verify you see in a large browser window:
```
.su-wysiwyg-text p {
    font-size: 2rem;
       : 
}
```
and in a smaller browser window:
```
@media (max-width: 767px)
.su-wysiwyg-text p {
    font-size: 1.6rem;
}
```

# Affected Projects or Products 
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# QA
## HTML Validation Results (https://validator.w3.org/)
n/a
## Accessibility Testing 
n/a

## Cross-Browser testing:
What browsers have been tested?
- Chrome Desktop - Yes
- Chrome Android
- FireFox Desktop
- FireFox Android
- OSX Safari
- iOS Safari
- Edge
- IE 11

## Unit Testing
N/A

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-1177

## Related PRs


## More Information

## Folks to notify
`@sherakama` 
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
